lib/rataextra-pipeline.ts

```diff
@@ -40,6 +41,37 @@ export class RataExtraPipelineStack extends Stack {
         },
       },
     });
+
+    const strip = new CodeBuildStep('StripAssetsFromAssembly', {
+      input: pipeline.cloudAssemblyFileSet,
+      commands: [
+        "cross_region_replication_buckets=$(grep BucketName cross-region-stack-* | awk -F 'BucketName' '{print $2}' | tr -d ': ' | tr -d '\"' | tr -d ',')",
+        'S3_PATH=${CODEBUILD_SOURCE_VERSION#"arn:aws:s3:::"}',
+        'ZIP_ARCHIVE=$(basename $S3_PATH)',
+        'rm -rf asset.*',
+        'zip -r -q -A $ZIP_ARCHIVE *',
+        'aws s3 cp $ZIP_ARCHIVE s3://$S3_PATH',
+        'object_location=${S3_PATH#*/}',
+        'for bucket in $cross_region_replication_buckets; do aws s3 cp $ZIP_ARCHIVE s3://$bucket/$object_location; done',
+      ],
+      rolePolicyStatements: [
+        new PolicyStatement({
+          effect: Effect.ALLOW,
+          resources: ['arn:aws:s3:::*'],
```

Comment:
Definitely better, but there's still room for improvement later on this. Maybe add a TODO here for when there's need to update the bucket to a removable one that this should also be updated then.

Reply:
Adding this TODO